docs/_docs/memory-configuration/replacement-policies.adoc

```diff
@@ -0,0 +1,96 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+= Replacement Policies
+
+When link:persistence/native-persistence[Native Persistence] is on and Ignite stores on disk more data than off-heap memory allocated for the data region, to preload some page from the disk to the completely full off-heap memory another page should be evicted from the off-heap to the disk. This process is called _page rotation_ or _page replacement_.
+
+When Native Persistence is off _eviction_ is used instead of _page replacement_. See link:memory-configuration/eviction-policies[Eviction Policies] page for more information.
```

Comment:
```suggestion
When Native Persistence is off, _eviction_ is used instead of _page replacement_. See the link:memory-configuration/eviction-policies[Eviction Policies] page for more information.
```